runtime/gc_vlhgc/AllocationContextBalanced.cpp

```diff
@@ -689,40 +689,42 @@ MM_AllocationContextBalanced::acquireFreeRegionFromHeap(MM_EnvironmentBase *env)
 }
 
 MM_HeapRegionDescriptorVLHGC *
-MM_AllocationContextBalanced::acquireMPBPRegionFromNode(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocationContextTarok *requestingContext)
+MM_AllocationContextBalanced::acquireMPAOLRegionFromNode(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocationContextTarok *requestingContext)
```

Comment:
name of this method cold be generic acquireMPRegionFromNode (not refer to AOL or BP)

Reply:
same for acquireMPAOLRegionFromHeap (drop AOL)